config/zones.json

```diff
@@ -771,7 +771,11 @@
       "price": "ENTSOE.fetch_price",
       "production": "ENTSOE.fetch_production"
     },
-    "timezone": null
+    "timezone": null,
+    "delays": {
+      "production": 24,
+      "price": 8
+    }
```

Comment:
Belarus actually doesn't have delayed data, but there rather seems to be a bug in the parser.

If you want an example, then you can add the following for `US-TEX-ERCO`

```
    "delays": {
      "production": 30
    }
```